export/cloud_datastore_admin.py

```diff
@@ -52,9 +54,9 @@ def get(self):
         """
         gcs_bucket_url_prefix = 'gs://'
 
-        access_token, _ = app_identity.get_access_token(
+        access_token, _ = app_identity_services.get_access_token(
```

Comment:
This is wrong, right? You don't need the ", _"

Also, is there not a unit test that would have caught this? If not, perhaps it's worth adding one.

Reply:
Yes you're right, looks like we don't have coverage for this file... I'll fix the error in this PR (great catch!) and will try to figure out why it isn't covered in a follow-up PR (because I think we should make all gathering PRs "rollback-able")

Reply:
Per discussion: let's cover it in a separate PR, merge that, get this PR on top of it, make sure the backend tests pass, and then merge this PR.

Reply:
Will be resolved by #10800

Reply:
Ditto.